airflow/providers/hashicorp/secrets/vault.py

```diff
@@ -189,7 +189,7 @@ def _get_secret(self, path_prefix: str, secret_id: str) -> Optional[dict]:
                 response = self.client.secrets.kv.v2.read_secret_version(
                     path=secret_path, mount_point=self.mount_point)
         except InvalidPath:
-            self.log.info("Secret %s not found in Path: %s", secret_id, secret_path)
+            self.log.error("Secret %s not found in Path: %s", secret_id, secret_path)
```

Comment:
```suggestion
            self.log.debug("Secret %s not found in Path: %s", secret_id, secret_path)
```

This one is expected, so change it to `debug`.

Reply:
Thanks, good catch!